editor/src/core/shared/element-path-tree.ts

```diff
@@ -133,6 +134,73 @@ export function reorderTree(trees: ElementPathTrees, metadata: ElementInstanceMe
   }
 }
 
+function maybeReorderDynamicChildren(
+  children: ElementPathTree[],
+  metadata: ElementInstanceMetadataMap,
+): ElementPathTree[] {
+  function keepOnlyUnique(path: ElementPath, index: number, array: ElementPath[]) {
+    return array.findIndex((e) => EP.pathsEqual(path, e)) === index
+  }
+
+  // Get all the dynamic children and sort them
+  // alphabetically, since their paths will have incremental indexes (~~~N).
```

Comment:
won't this fail if there's more than 10 dynamic children? ie if we are comparing `aaa~~~2` with `aaa~~~12`?

Reply:
I just quickly checked and it looks like that's handled by `localeCompare`:

```
> '~~~12'.localeCompare('~~~1')
< 1
> '~~~2'.localeCompare('~~~1')
< 1
```